spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -582,6 +583,53 @@ func TestCreateDatabaseWithDefaultLeaderSample(t *testing.T) {
 	assertContains(t, out, "The result of the query to get")
 }
 
+func TestUpdateDatabaseSample(t *testing.T) {
+	_ = testutil.SystemTest(t)
+	t.Parallel()
+
+	_, dbName, cleanup := initTest(t, randomID())
+	defer cleanup()
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Hour)
```

Comment:
This seems too long; do you expect the operation to actually take an hour?

Reply:
@tritone 
We have been using 60 minutes as a standard for all the existing tests, so following the same here also.
Ref: https://github.com/GoogleCloudPlatform/golang-samples/blob/main/spanner/spanner_snippets/spanner/integration_test.go#L136

Reply:
Okay got it, no problem if this works for you but something to keep an eye on if you start getting test hangs/timeouts.